portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/BalloonEditor.testcase

```diff
@@ -5,12 +5,23 @@ definition {
 	property portal.upstream = "true";
 	property testray.main.component.name = "User Interface";
 	property portal.acceptance = "true";
-	property osgi.modules.includes = "frontend-editor-ckeditor-sample-web";
```

Comment:
unfortunately adding this line only is not working

Reply:
Hmm...weird, I'll see if I can get one to start on my end.

Reply:
Testing here: 
1. https://github.com/john-co/liferay-portal/pull/414 - base
2. https://github.com/john-co/liferay-portal/pull/415 - see if need to add module to module.properties

Reply:
Both tests passed: with the osgi.modules.includes property on ci

So it will work on ci. My guess for why it may not be working for you was, if you're testing locally you'll want to run: `run-selenium-tomcat` instead of `run-simple-server` to properly deploy the osgi module.

Reply:
thanks for checking! will remove the unnecessary lines. however, none of the commands you've mentioned sound familiar, when exactly do you use them?

Reply:
How did you verify the test wasn't working for you with `property osgi.modules.includes = "frontend-editor-ckeditor-sample-web";`?

I was assuming you were running the poshi test locally on your machine. The command to run a poshi test locally is: `ant -f build-test.xml run-selenium-test -Dtest.class="$1"` 

If the test needs to deploy something because it has a osgi.modules.includes property, then I suggest trying: `ant -f build-test.xml run-selenium-tomcat -Dtest.class="$1"` so the test can properly pick up that property and deploy the module.

Reply:
so it would look like: `ant -f build-test.xml run-selenium-tomcat -Dtest.class="BalloonEditor#BalloonToolbarTextEditOptions"`